download.sh

```diff
@@ -27,14 +27,26 @@ rm /opt/gpu/${RUNFILE}.run
 popd
 
 
+install_fabric_manager () {
+    local driver_ver="${DRIVER_VERSION}"
+    if [[ "${DRIVER_VERSION}" == "535.161.08" ]]; then
+      local driver_ver="535.104.05"
+      # Temporary as latest fabric manager version here is not the same as the driver version. https://developer.download.nvidia.com/compute/cuda/redist/fabricmanager/linux-x86_64/
```

Comment:
can we add a TODO item for this?

Reply:
Do you mean adding a "TODO" in the comment or creating a work item?
Depending on whether fabric manager has a new version matching the driver, this will need to be changed

Reply:
TODO with a reference to the ADO work item would be best. 

``` 
// TODO decouple the fabric_manager version from the driver version : ado# 1234
```

Reply:
Ok. Note - ADO work item # will be publicly accessible, though that should be ok.

Reply:
ohhh that's a public repo... so I guess GIT issue.